doc/rule-descriptions.md

```diff
@@ -34,6 +34,8 @@
 | label-title-only | Ensures that every form element is not solely labeled using the title or aria-describedby attributes | cat.forms, best-practice | true |
 | label | Ensures every form element has a label | cat.forms, wcag2a, wcag332, wcag131, section508, section508.22.n | true |
 | landmark-main-is-top-level | The main landmark should not be contained in another landmark | best-practice | true |
+| landmark-no-duplicate-banner | A banner landmark identifies site-oriented content at the beginning of each page within a website | best-practice | true |
```

Comment:
It looks like most of the rule description read as descriptions of what the rule checks for, not as descriptions of the benefit of a given accessibility feature. We should be consistent. This would become something like, "Ensures the document has no more than one banner landmark."